github-event-service/src/firebase-init.ts

```diff
@@ -0,0 +1,14 @@
+import * as firebase from 'firebase';
+
+const config = {
+  // TODO(josephperrott): Set up config via ENV variables.
+};
+
+/** The initialized Firebase app instance */
+let firebaseApp: firebase.app.App;
+
+/** Gets the active instance of the Firebase app, creating one if necessary. */
+export const getFirebaseInst = (): firebase.app.App => {
```

Comment:
`Inst` seems a bit odd. How about just writing `getFirebaseInstance`? Also setting a return would be easier if it's just a normal function.